app/gather/assets/css/_survey-edit.scss

```diff
@@ -114,7 +114,7 @@ input.hidden-file {
   }
 }
 
-.FilteredMultiSelect { // sass-lint:disable-line class-name-format
+.FilteredMultiSelect {
```

Comment:
Actually I think this rule is good :D - I think we should keep it

Reply:
Reintroduce :wink: